otx/cli/builder/builder.py

```diff
@@ -301,7 +275,15 @@ def build_model_config(self, model_config_path, backbone_config_path, output_pat
 
         # Build Backbone
         backbone = build_from_cfg(backbone_config, otx_registry, None)
-        out_channels = get_backbone_out_channels(backbone)
+        if model_config.model.get("task", None) == "classification":
+            # Update model layer's in/out configuration in ClsStage.configure_model
+            out_channels = -1
+            if hasattr(model_config.model, "head"):
+                model_config.model.head.in_channels = -1
+        else:
```

Comment:
Maybe, we need to consider Action tasks, in the near future.
So, I just prefer the `elif` sentence in here to handle the `detection` and `segmentation`

Reply:
yes, I think this is something to consider. I will test it and if it is not a problem, I will fix it with `elif` & `else` (raise Error).

Reply:
As a result of checking this, there are cases where there is no task variable for each model. Looks like I need to align this later. So I think we should leave it as it is for now.